src/External/CMakeLists.txt

```diff
@@ -173,13 +173,22 @@ list(APPEND EXTERNAL_INCLUDE_DIRS
      ${GLEW_INCLUDE_DIRS}
      ${GLFW_INCLUDE_DIRS}
      ${JPEG_INCLUDE_DIR}
```

Comment:
Do we need to remove ``JPEG_INCLUDE_DIR``?

Reply:
My hypothesis is that for some versions of libjpeg it returns JPEG_INCLUDE_DIR for other versions it returns JPEG_INCLUDE_DIRS.